types/testingutils/runner.go

```diff
@@ -276,3 +277,42 @@ var DecidingMsgsForHeightWithRoot = func(root [32]byte, fullData, msgIdentifier
 	}
 	return msgs
 }
+
+var ExpectedDecidingMsgsForHeightWithRoot = func(root [32]byte, fullData, msgIdentifier []byte, height qbft.Height, keySet *TestKeySet) []*qbft.SignedMessage {
```

Comment:
can you explain the difference between `DecidingMsgsForHeightWithRoot` and `ExpectedDecidingMsgsForHeightWithRoot`?